themes/openy_themes/openy_rose/scss/modules/_branch.scss

```diff
@@ -985,3 +982,65 @@ html.js {
     margin-right: 0.25em;
   }
 }
+
+.marker_tooltip {
```

Comment:
no need to add code to every theme, use global include https://github.com/ymcatwincities/openy/blob/8.x-1.13-dev/themes/openy_themes/scss/modules.scss to include your scss/sass file from all themes available in a system, - just recompile them all in case if there is a change

Reply:
@podarok Has the change been announced?
What if the module file is going to have mixin calls within it?